tools/data-handler/src/create.ts

```diff
@@ -408,6 +408,23 @@ export class Create extends EventEmitter {
     await this.createResource('fieldType', content);
   }
 
+  public async createLabel(cardKey: string, label: string) {
```

Comment:
'label' should follow generic naming rules.
At least not empty string, must be at least N characters long and no longer than M characters. 
Should not contain extra characters.
There's at least `isValidResourceName()` function that could be used. It was intended for resource identifiers, but labels could follow similar rules.